e2etest/bgp_test.go

```diff
@@ -150,6 +152,14 @@ var _ = ginkgo.Describe("BGP", func() {
 					Name:      "bgp-test",
 					Protocol:  BGP,
 					Addresses: poolAddresses,
+					BGPAdvertisements: []bgpAdvertisement{
```

Comment:
I'd rather have a single new tests with the communities (and in general, were we shuffle the parameters) than extending all the tests with communities so we make sure the basic use case (all default) works.

Reply:
added specific test case for community testing